Source/Scene/TileBoundingVolume.js

```diff
@@ -0,0 +1,43 @@
+/*global define*/
+define([
+        '../Core/DeveloperError'
+    ], function(
+        DeveloperError) {
+    "use strict";
+
+    /**
+     * Defines a bounding volume for a tile. This type describes an interface
+     * and is not intended to be instantiated directly.
+     *
+     * @see TileBoundingBox
+     * @see TileBoundingSphere
+     */
+    var TileBoundingVolume = function() {
+    };
+
+    /**
+     * Calculates the distance between the tile and the camera.
+     *
+     * @param {FrameState} frameState The frame state.
+     * @return {Number} The distance between the tile and the camera, in meters.
+     *                  Returns 0.0 if the camera is inside the tile.
+     */
+    TileBoundingVolume.prototype.distanceToCamera = function(frameState) {
+        DeveloperError.throwInstantiationError();
+    };
```

Comment:
I changed the instantiation errors to follow the convention in `Cesium3DTileContentProvider`, but I realize most of Cesium code base does it the other way.